src/main/java/com/codecool/terraformingmarsforum/service/CommentService.java

```diff
@@ -0,0 +1,13 @@
+package com.codecool.terraformingmarsforum.service;
+
+import com.codecool.terraformingmarsforum.service.DAO.CommentDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CommentService {
+
+    @Autowired
+    private CommentDAO commentDAO;
```

Comment:
Again, field injection? (I think constructor injection is better because the field can be IMMUTABLE then)